src/java.desktop/macosx/classes/com/apple/laf/resources/aqua_ja.properties

```diff
@@ -60,6 +60,7 @@ FileChooser.saveTitle.textAndMnemonic=\u4FDD\u5B58
 FileChooser.openTitle.textAndMnemonic=\u958B\u304F
 FileChooser.newFolderExistsError.textAndMnemonic=\u305D\u306E\u540D\u524D\u306F\u3059\u3067\u306B\u4F7F\u7528\u3055\u308C\u3066\u3044\u307E\u3059
 FileChooser.chooseButton.textAndMnemonic=\u9078\u629E
+FileChooser.chooseButtonToolTip.textAndMnemonic=\u9078\u629E\u3057\u305F\u30D5\u30A1\u30A4\u30EB\u3092\u8FFD\u52A0\u3057\u307E\u3059
```

Comment:
The translation seems wrong. It means to *add* the selected file, which is not the same meaning as in English file